contents/common.py

```diff
@@ -173,6 +173,48 @@ def log_pod_parameters(logger, data):
     logger.debug("--------------------------")
 
 
+def resolve_container_for_pod(name, namespace):
+    core_v1 = client.CoreV1Api()
+    response = core_v1.read_namespaced_pod_status(
+        name=name,
+        namespace=namespace,
+        pretty="True"
+    )
+
+    if response.spec.containers:
+        container = response.spec.containers[0].name
+        return container
+    log.error("Container not found for pod %s", name)
+    exit(1)
+
+
+def get_active_pods_for_deployment(name, namespace):
+    """
+    Retrieve all pods belonging to a deployment
+    """
+    api = core_v1_api.CoreV1Api()
+    resp = None
+    try:
+        resp = api.list_namespaced_pod(namespace=namespace)
+    except ApiException as e:
+        if e.status != 404:
+            log.exception("Unknown error:")
+            exit(1)
+
+    if not resp:
+        log.error("Namespace %s does not exits.", namespace)
```

Comment:
spelling: "exits" should be "exist"

Reply:
Thanks for the review, I fixed the spelling mistake :)